python/mxnet/image.py

```diff
@@ -351,8 +351,8 @@ class ImageIter(io.DataIter):
     data_shape : tuple
         Data shape in (channels, height, width) format.
         For now, only RGB image with 3 channels is supported.
-    label_width : int
-        Label dimension.
+    label_width : int, optional
+        Label dimension. The default label width is 1.
```

Comment:
Does this mean the number of classes? Perhaps we can be clearer? Right now we say "Label dimension" and "label width" ostensibly to refer to the same quantity. Let's use consistent notation.